vmdb/app/models/ems_refresh/parsers/openshift.rb

```diff
@@ -14,16 +14,20 @@ def get_routes(inventory)
       end
 
       def get_projects(inventory)
-        process_collection(inventory["project"], :container_projects) { |n| parse_project(n) }
+        process_collection(inventory["project"], :container_projects_openshift) { |n| parse_project(n) }
```

Comment:
I think this should become simply:

    inventory["project"].each { |item| parse_project(item) }

Since this part won't be actually returning any new data, it would just augment the data already collected.

/cc @Fryguy